README.md

```diff
@@ -355,6 +355,8 @@ You can add CombineSchedulers to an Xcode project by adding it as a package depe
       - If you have a single application target that needs access to the library, then add **CombineSchedulers** directly to your application.
       - If you want to use this library from multiple targets you must create a shared framework that depends on **CombineSchedulers**, and then depend on that framework from your other targets.
 
+**Note**: This library is only compatible with iOS 13.2 and higher. There are bugs in Combine and iOS 13.1 and lower that cause crashes when trying to compare `DispatchQueue.SchedulerTimeType` values.
```

Comment:
What do you think of moving this to a `## Compatibility` section above the installation? Either that or moving this into a `>` quote with a ⚠️ moj?

Reply:
ah yeah that works for me